src/libraries/Microsoft.PowerFx.Core/Texl/Builtins/Unichar.cs

```diff
@@ -0,0 +1,76 @@
+﻿// Copyright (c) Microsoft Corporation.
+// Licensed under the MIT license.
+
+using System.Collections.Generic;
+using Microsoft.PowerFx.Core.App.ErrorContainers;
+using Microsoft.PowerFx.Core.Errors;
+using Microsoft.PowerFx.Core.Functions;
+using Microsoft.PowerFx.Core.Functions.Delegation;
+using Microsoft.PowerFx.Core.Localization;
+using Microsoft.PowerFx.Core.Types;
+using Microsoft.PowerFx.Core.Utils;
+using Microsoft.PowerFx.Syntax;
+
+namespace Microsoft.PowerFx.Core.Texl.Builtins
+{
+    // Unichar(arg:n) : s
+    // Corresponding Excel function: Unichar
+    internal sealed class UnicharFunction : BuiltinFunction
+    {
+        public override ArgPreprocessor GetArgPreprocessor(int index, int argCount)
+        {
+            return base.GetGenericArgPreprocessor(index);
+        }
+
+        public override bool IsSelfContained => true;
+
+        public override bool IsStateless => true;
+
+        public UnicharFunction()
+            : base("Unichar", TexlStrings.AboutUnichar, FunctionCategories.Text, DType.String, 0, 1, 1, DType.Number)
```

Comment:
>Unichar [](http://example.com/codeflow?start=20&length=7)

Should we use `UniChar`? This is a compound noun ("Unicode Char"), and for other functions we use pascal casing (e.g., LookUp, AddColumns, TimeValue, IsMatch, and so on)

Reply:
sounds good to me